collect_app/src/main/res/layout/arbitrary_file_widget_answer_view.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content">
+
+    <ImageView
+        android:id="@+id/icon"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:tint="?colorOnSurface"
+        app:srcCompat="@drawable/ic_baseline_attach_file_white_24"
+        app:layout_constraintTop_toTopOf="@id/answer"
+        app:layout_constraintBottom_toBottomOf="@id/answer"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/answer"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:tag="ArbitraryFileWidgetAnswer"
```

Comment:
Why do we need this tag?

Reply:
It is used in tests to find this type of question, and for example, to check if it is displayed.